src/app/home-page/widget/organizations/organizations.component.html

```diff
@@ -16,7 +16,7 @@ <h3>My Organizations</h3>
     </mat-form-field>
 
     <div *ngFor="let organization of myItems" class="w-100 ellipsis-overflow">
-      <span [matTooltip]="organization.name">
+      <span [matTooltip]="organization.displayName">
```

Comment:
Is displayName always set? If not, should it be organization.displayName || organization.name?

Reply:
It is a required field so it will always be set.